src/riak_governor_ensemble_master.erl

```diff
@@ -75,7 +89,12 @@ ensure_ensembles_started(#state{ensemble_size=EnsembleSize}) ->
     Ensembles0 = all_ensembles(EnsembleSize),
     ClusterNodes = riak_governor_util:get_cluster_nodes(),
     Ensembles = [ClusterNodes|Ensembles0],
-    [ensure_ensemble_started(Nodes) || Nodes <- Ensembles].
+    Res = [ensure_ensemble_started(Nodes) || Nodes <- Ensembles],
+    % validate all results are ok
+    case lists:usort(Res) of
+        [ok] -> ok;
+        _ -> {error, {unexpected_ensemble_start_results, Res}}
+    end.
```

Comment:
Looking at the chain of function calls, I can only see outcome of ok | {error, timeout} | {error, failed}. This is also supported by manual testing.  Can you see evidence of otherwise?
riak_ensemble_manager:create_ensemble()
riak_ensemble_root:set_ensemble()
riak_ensemble_root:call()
riak_ensemble_peer:kmodify()
riak_ensemble_router:sync_send_event()
riak_ensemble_router:sync_proxy()
riak_ensemble_router:sync_proxy_direct() or riak_ensemble_router:sync_proxy_router()
riak_ensemble_router:do_kmodify()
riak_ensemble_root:do_root_call()